playbooks/openshift-web-console/private/redeploy-certificates.yml

```diff
@@ -1,35 +1,48 @@
 ---
+
 - name: Update web console certificates
   hosts: oo_first_master
   vars:
   roles:
   - lib_openshift
   - openshift_facts
   tasks:
-  - name: Remove certificates secret
-    oc_obj:
-      name: webconsole-serving-cert
-      kind: secret
-      state: absent
-      namespace: openshift-web-console
+  - name: Remove certificates secret in openshift-web-console
+      shell: oc delete secret console-serving-cert -n openshift-console
 
   - name: Remove web console pods
-    oc_obj:
-      selector: "webconsole=true"
-      kind: pod
-      state: absent
-      namespace: openshift-web-console
+      shell: oc delete pods --all -n openshift-console
 
   - name: Verify that the console is running
     oc_obj:
-      namespace: openshift-web-console
+      namespace: openshift-console
```

Comment:
There are two consoles in 3.11. Which console are you meaning to update? This playbook is for the developer console in the openshift-web-console namespace.